MIGRATING_TO_V8.0.0.MD

```diff
@@ -0,0 +1,214 @@
+- Before 8.0.0: `rqlBuilder().eq('diagnosis', encodeURIComponent('Hypertension - STAGE 1'))`
+- After 8.0.0: `rqlBuilder({ doubleEncodeValues: false }).eq('diagnosis', encodeURIComponent('Hypertension - STAGE 1'))`
+
+### Reverting double encoding for all rql builders
+The automatic double encoding of values can be reverted to behavior before 8.0.0 using the snippet `rqlBuilder.doubleEncodeValues = false`. This should normally exist alongside the declaration of your Extra Horizon SDK instance at the root of your application.
+- You may then enable double encoding for an rql builder `rqlBuilder({ doubleEncodeValues: true }).eq('diagnosis', 'Hypertension - STAGE 1')`
+- You may manually encode values for an rql builder `rqlBuilder().eq('diagnosis', encodeURIComponent('Hypertension - STAGE 1'))`
+
+For more information regarding manual double encoding please refer to the [RQL documentation](https://docs.extrahorizon.com/extrahorizon/additional-resources/resource-query-language-rql#double-encoding-of-special-characters).
+
+## Custom data normalization
+
+From version 8.0.0, the SDK will no longer normalize custom keys in requests and responses. This means that all custom keys will be sent and received as they are provided.
+
+Before version 8.0.0 all keys were normalized to snake_case in requests and camelCase in responses. This is useful because some of our services still return snake_case keys, while others return camelCase keys. 
+We wanted to keep this behaviour for all keys defined by Extra Horizon services, but we also wanted to allow custom keys to be sent and received as they are provided.
+
+### Changes to request data
+If you are providing custom keys in camel case before v8.0.0, they will now be stored as camel case.
+While before they would be stored as snake case for some requests.
+
+For all [affected properties](###-affected-properties) with the request mentioned in v8.0.0 the custom data in the request will now be stored as it was provided.
+
+Before v8.0.0 some custom data in requests was transformed to snake case:
+```js
+// Setting the custom fields of a profile with a camel cased custom key
+await exh.profiles.update(profile.id, {
+    customFields: {
+        helloWorld: 'test',
+    },
+});
+
+// Before v8.0.0 the customFields of the profile would be stored as
+// { 
+//     hello_world: 'test' 
+// }
+
+// After v8.0.0 the customFields of the profile would be stored as
+// {
+//     helloWorld: 'test'
+// }
+```
+
+### Changes to response data
+For all [affected properties](###-affected-properties) with the response mentioned in v8.0.0 the response will be returned as it was provided.
+
+#### Camel Casing Custom Keys
+
+If you are providing custom keys in snake case, you will now receive them as snake case in the responses as well.
+While before you would receive them as camel case.
+
+Before v8.0.0 some responses used to be camel cased:
```

Comment:
I find this title a little misleading and it could be just left out for me? 🤔 

It implies the code block is regarding only the old behaviour rather than the difference. If I was a developer reading this I might skip this block if I was looking for only the new responses.